File: javascript/Q8.js
// Find the Average of an Array. Example: Input: arr=[1,2,3,4,5]; Output: 3
let arr=[1,2,3,4,5]
let len=arr.length
let i=0;
let sum=0;
let avg;
for(i=0;i<len;i++){
    sum=sum+arr[i]
    avg=sum/len
}

console.log(avg)

File: c lang/Q8.c
// 8. Find the Average of an Array.
# include <stdio.h>
int main(){
	int arr[]={1,2,3,4,5};
	int i=0;
	int sum;
	int len=sizeof(arr)/sizeof(arr[0]);
	for(i=0;i<len;i++){
		sum=sum+arr[i];
		int avg=sum/len;
	
		printf("%d",avg);
	}
		
}
